.github/workflows/main.yml

```diff
@@ -382,18 +382,12 @@ jobs:
       # This is a workaround from: https://github.com/actions/checkout/issues/590#issuecomment-970586842
       - run: "git checkout -f $(git -c user.name=x -c user.email=x@x commit-tree $(git hash-object -t tree /dev/null) < /dev/null) || :"
       - uses: actions/checkout@v4
-        with:
-          lfs: true
       - uses: ./.github/actions/rustup
-      - uses: ./.github/actions/sccache
-        with:
-          key: Linux-default
```

Comment:
This is useful for speeding up the `cargo run --bin cargo-risczero` and also building the tests. I don't think we should drop this.

Reply:
I've reverted this